libs/ui/molecules/src/lib/AuctionHearts/TextStyled/index.tsx

```diff
@@ -0,0 +1,8 @@
+import styled from 'styled-components/native';
+import { Text } from '@inheartive/ui/atoms';
+
+const TextStyled = styled(Text)`
+  top: -2;
```

Comment:
Why is it needed? If to place text in the middle, would vertical-align work or something similar?